diesel/src/associations/mod.rs

```diff
@@ -1,3 +1,113 @@
+//! have been released. Please feel free to open issues, or join [our chat][gitter] to provide
+//! feedback.**
+//!
+//! [gitter]: https://gitter.im/diesel-rs/diesel
+//!
+//! Note: This guide is written assuming the usage of `diesel_codegen` or `diesel_codegen_syntex`.
+//! If you are using [`custom_derive`] instead, you will need to replace `#[belongs_to(Foo)]` with
+//! `#[derive(BelongsTo(Foo, foreign_key = foo_id))]` and `#[has_many(bars)]` with
+//! `#[derive(HasMany(bars, foreign_key = bar_id))]`.
+//!
+//! Associations in Diesel are bidirectional, but primarily focus on the child-to-parent
+//! relationship. You can declare an association between two records with `#[has_many]` and
+//! `#[belongs_to]`.
+//!
+//! ```ignore
+//! #[derive(Identifiable, Queryable)]
+//! #[has_many(posts)]
+//! pub struct User {
+//!     id: i32,
+//!     name: String,
+//! }
+//!
+//! #[derive(Identifiable, Queryable)]
+//! #[belongs_to(User)]
+//! pub struct Post {
+//!     id: i32,
+//!     user_id: i32,
+//!     title: String,
+//! }
+//! ```
+//!
+//! `#[has_many]` shoudld be passed the name of the table that the children will be loaded from,
+//! while `#[belongs_to]` is given the name of the struct that represents the parent. Both types
+//! must implement the [`Identifiable`][identifiable] trait.
+//!
+//! [Identifiable]: associations/trait.Identifiable.html
+//!
+//! `#[has_many]` actually has no behavior on its own. It only enables joining between the two
+//! tables. If you are only accessing data through the [`belonging_to`][belonging-to] method, you
+//! only need to define the `#[belongs_to]` side.
+//!
+//! Once the associations are defined, you can join between the two tables using the
+//! [`inner_join`][inner-join] or [`left_outer_join`][left-outer-join].
+//!
+//! [inner-join]: query_source/trait.Table.html#method.inner_join
+//! [left-outer-join]: query_source/trait.Table.html#method.left_outer_join
+//!
+//! ```ignore
+//! let data: Vec<(User, Post)> = users::table.inner_join(posts::table).load(&connection);
+//! ```
+//!
+//! Note: Due to language limitations, only two tables can be joined per query. This will change in
+//! the future.
+//!
+//! Typically however, queries are loaded in multiple queries. For most datasets, the reduced
+//! amount of duplicated information sent over the wire saves more than the the extra round trip
+//! costs us. You can load the children for a single parent using the
+//! [`belonging_to`][belonging-to]
+//!
+//! [belonging-to]: prelude/trait.BelongingToDsl.html#tymethod.belonging_to
+//!
+//! ```ignore
+//! let user = try!(users::find(1).first(&connection));
+//! let posts = Post::belonging_to(&user).load(&connection);
+//! ```
+//!
+//! If you're coming from most other ORMs, you'll notice that this design is quite different from
+//! most, where you would have an instance method on the parent, or have the children stored
```

Comment:
Two "most", get rid of one.

I'd also split this into two sentences: `s/, where/. There,`